src/ts/Search/Select.tsx

```diff
@@ -1,134 +1,166 @@
 import * as React from 'react';
-import { FontIcon } from 'material-ui';
-import { default as SuperSelect } from 'material-ui-superselectfield';
+import * as _ from 'lodash';
+import {
+  FontIcon,
+  SelectField,
+  MenuItem,
+  Toggle,
+} from 'material-ui';
 import debug from 'debug';
 const log = debug('app:Select');
 import i from '../i18n';
 
 export default class Select extends React.Component<any, any> {
-  private dataSource = [];
-  constructor(props) {
-    super(props);
-    // select keeps values chosen by user
-    this.state = { value: [] };
-    this.onChange = this.onChange.bind(this);
+  state = {
```

Comment:
setting the state in the constructor is the recommended way: https://facebook.github.io/react-native/docs/state.html

I understand that they MAY be equivalent (?), but let's just keep a single way of handling things please